cpp/daal/src/services/compiler/generic/env_detect_features.cpp

```diff
@@ -193,7 +211,7 @@ static int check_sse42_features()
 
 DAAL_EXPORT bool __daal_serv_cpu_extensions_available()
 {
-    return daal_check_is_intel_cpu();
+    return 1;
```

Comment:
First, if a bool, it would be best to return a bool. Secondly, if this is a no-op, then the function should be removed.

Reply:
I removed the function and its usages.